dev/readthedocs.md

```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
+   python manage.py ingest jhu --years 2021 --terms fall
 
 2. How do I then load those courses into my database?
+   python manage.py digest jhu --years 2021 --terms fall
 
 3. How do I get a terminal running in my docker container?
+   docker-compose up
 
 4. Where do I store data I don’t want to commit?
+   add to .gitignore
 
 5. What is our stack?
+   Our stack includes Database, Backend and Frontend Frameworks, and CSS Framework. 
 
 6. What branch do I create a new branch off of when developing?
+   develop
 
 7. If I want to start on a feature called myfeature, what should the branch name be?
+   feature/myfeature
 
    What about if I want to refactor myreducer?
+      refactor/refactor-myreducer
 
 8. What is the preferred format for commit messages?
+   Topic: Message
+   e.g.: 
+      Refactor Redux: refactor myreducer to myslice
 
 9. What linters do we run against our code?
+   ESLint
```

Comment:
There's two more, one more for frontend, and one for backend. What are they?

Reply:
Frontend uses ESlint and Prettier, and Backend uses pycodestyle/Black.

Reply:
Correct me if Im wrong but are these like the IntelliJ CheckStyle to make code neat and consistent? Is there anything in particilar we should consider while writing code, or should we be running the linters in the end and make adjustments accordingly?

Reply:
Yes, these are like CheckStyle and ensures the code is consistent and standardly formatted. When writing code, we try to be as clear as possible and write clean code. If you want to learn more, there's a well known book conveniently called Clean Code that I recommend (first 10 chapters or so)

Reply:
Thanks. Will check this out when I have time!